### DISCUSSION TITLE
Cloudfoundry Application logs

### DISCUSSION THREAD

**Discussion by manuvnair** [vectordotdev/vector#22096] [Q&A] [answered]
How can I send Cloudfoundry Application logs to Elasticsearch using Vector?

**Comment by bfung** [comment, marked as answer]
You can try to configure logging in Cloudfoundry to drain to a file or stream, https://docs.cloudfoundry.org/devguide/services/log-management.html, and then use File, FileDescriptor, syslog, or pipe to stdin sources.  Then transform and pipe to Elasticsearch sink.
